app.js

```diff
@@ -14,3 +14,8 @@ app.use(express.static('public'));
 app.use('', postController);
 
 app.listen(3000);
+<<<<<<< HEAD
+=======
+
+module.exports = app;
+>>>>>>> 890ea25... add tentative postController tests using supertest
```

Comment:
??? fix this, don't commit this

Reply:
fixed

Reply:
Doesn't look fixed to me, did you push your changes?
Honestly, you don't even need to comment fixed, just push your changes